cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```
@@ -574,6 +609,8 @@ objects:
           - --azure-first-party-application-client-id=${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID}
           - --azure-first-party-application-certificate-bundle-path=/secrets/keyvault/firstPartyApplicationCertificateBundle
           - --azure-runtime-config-path=/configs/azure-runtime-config/config.json
+          - --arm-helper-identity-client-id=${ARM_HELPER_IDENTITY_CLIENT_ID}
+          - --arm-helper-identity-certificate-bundle-path=/secrets/arm-identity/armHelperIndentityCertificateBundlePath
```

```suggestion
          - --arm-helper-identity-certificate-bundle-path=/secrets/arm-identity/armHelperIndentityCertificateBundle
```
You have not specified this name.

---

added